cosmos/providers/dbt/core/operators/tests/test_local.py

```diff
@@ -83,3 +83,36 @@ def test_dbt_base_operator_get_env(p_context_to_airflow_vars: MagicMock) -> None
         "START_DATE": "2023-02-15 12:30:00",
     }
     assert env == expected_env
+
+
+@patch("os.path.exists")
+def test_get_profile_name(mock_os_path_exists) -> None:
+    mock_os_path_exists.return_value = True
+
+    # check that a user-specified profile name is returned when specified
+    dbt_base_operator = DbtLocalBaseOperator(
+        conn_id="my_airflow_connection",
+        task_id="my-task",
+        project_dir="my/dir",
+        profile_name="default",
```

Comment:
It may be worth giving a different name since "default" is already the default behaviour if this parameter isn't given